### PR TITLE
Cleanup nfs-share-creator

### PR DESCRIPTION
**Pull request by yuvipanda**
https://pilot-hubs.2i2c.org/en/latest/topic/storage-layer.html
has more info on the nfs-share-creator. This PR adds support for
setting baseSharePath to `/`, which is sometimes needed on EFS.
    
Ref https://github.com/2i2c-org/pilot-hubs/issues/366

### CODE REVIEW

File: hub-templates/base-hub/templates/nfs-share-creator.yaml [hunk 1]
```diff
@@ -31,10 +31,12 @@ spec:
           command:
             - /bin/sh
             - -c
-            - "mkdir -p ${NFS_SHARE_NAME} && chown 1000:1000 ${NFS_SHARE_NAME}"
+            # ${NFS_SHARE_NAME} starts with `/`
+            - "mkdir -p /mnt${NFS_SHARE_NAME} && chown 1000:1000 /mnt${NFS_SHARE_NAME} && mount && ls -la /mnt/ && sleep 60h"
```
Comment (damianavila) [inline hub-templates/base-hub/templates/nfs-share-creator.yaml:35]: Why the additional `&& mount && ls -la /mnt/ && sleep 60h` stuff?

Comment (yuvipanda) [reply hub-templates/base-hub/templates/nfs-share-creator.yaml:35]: Ah, the `sleep 60h` was for debugging and i'll take it out. The rest is for debugging - it took me forever to figure this out :D

Comment (damianavila) [reply hub-templates/base-hub/templates/nfs-share-creator.yaml:35]: I supposed it was debugging, this is why I asked about it :wink:

Comment (yuvipanda) [reply hub-templates/base-hub/templates/nfs-share-creator.yaml]: Fixed!

**Review by damianavila — APPROVED** [review]
LGTM

**Comment by yuvipanda** [comment]
Thanks!

**Pull request merged**